packages/client/src/object/FetchPageArgs.ts

```diff
@@ -20,14 +20,20 @@ import type {
   ObjectOrInterfaceDefinition,
   ObjectOrInterfacePropertyKeysFrom2,
 } from "@osdk/api";
+
+export type StrictNonNull = false | "throw" | "drop";
+export type StrictNonNullDefault = "throw";
```

Comment:
Is this safe to default to throw? This may break existing usage right, should this be opt-in instead of opt-out?

Reply:
Ok that's fair